server/api.js

```diff
@@ -8,5 +8,16 @@ router.get("/", (_, res) => {
 	logger.debug("Welcoming everyone...");
 	res.json({ message: "Hello, world!" });
 });
+router.get("/home", (req, res) => {
```

Comment:
why is the endpoint called `/home`?? this is going to return all pages so it should be something like `/pages` or `/getPages` or `/allPages`